tests/Unit/LoggerTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Laravel\Cashier\Tests\Unit;
```

Comment:
This test will need to be moved to the Integration suite because the framework is bootstrapped.